src/Persistent/Idiom.hs

```diff
@@ -91,7 +91,7 @@ checkInPhaseJury topic =
 checkAllIdeasMarked :: PersistM m => Topic -> m Bool
 checkAllIdeasMarked topic = all isMarkedIdea <$> findIdeasByTopic topic
```

Comment:
Rename this 'Marked' to 'Judged', and the winning-'Marked' as 'MarkedWinning'?  Either way I would like to have two names for these two things.